src/Ml.Cli.WebApp/Paths/ComparesPaths.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Ml.Cli.WebApp.Paths
+{
+    public class ComparesPaths
+    {
+        public string Paths { get; }
+        public ComparesPaths(string comparesPaths)
+        {
+            Paths = comparesPaths;
+        }
+
+        public bool IsPathContained(string filePath)
+        {
+            var paths = Paths.Split(',');
+            foreach (var path in paths)
```

Comment:
in one line => with an every function (+lambda)

Reply:
IsPathContained is now a lambda, oneline function